packages/opentelemetry-exporter-zipkin/src/transform.ts

```diff
@@ -84,7 +84,7 @@ export function _toZipkinTags(
   }
 
   Object.keys(resource.attributes).forEach(
-    name => (tags[name] = resource.attributes[name])
+    name => (tags[name] = String(resource.attributes[name]))
```

Comment:
A quick if-check would prevent a new string allocation in the case that the value is already a string.

```suggestion
    name => (tags[name] = typeof resource.attributes[name] === "string" ? resource.attributes[name] : String(resource.attributes[name]))
```

You could also simplify this by using `Object.entries`

```typescript
  for (const [name, value] of Object.entries(resource.attributes)) {
    tags[name] = typeof value === "string" ? value : String(value);
  }
```

Reply:
I see the pattern `String(...)` in dozens of places across the codebase, including many exporters (among them the zipkin exporter for which I wrote the fix) and could not find even one place where the type is checked before converting to `String`.
If you feel it's needed I will add it, but I think it's redundant.

Reply:
No its fine